client/extensions/ansa-archive/src/extension.ts

```diff
@@ -0,0 +1,44 @@
+import {IExtension, IExtensionActivationResult, ISuperdesk, IArticle, IArticleActionBulk} from 'superdesk-api';
+
+const isExternalPicture = (article: IArticle) =>
+    article._type === 'externalsource' &&
+    article.fetch_endpoint === 'search_providers_proxy';
+
+const extension: IExtension = {
+    id: 'ansa-archive',
+    activate: (superdesk: ISuperdesk) => {
+        const result: IExtensionActivationResult = {
+            contributions: {
+                entities: {
+                    article: {
+                        getActionsBulk: (articles) => {
+                            const actions: IArticleActionBulk[] = [];
```

Comment:
It'd be nice to stick to `Array<IArticleActionBulk>` format. It's enforced by a linter on the client, but not here.

Reply:
what's the difference?

Reply:
None, it's just a formatting choice.

Reply:
I see, ok makes sense to be consistent